.drone.yml

```diff
@@ -3,31 +3,95 @@ kind: pipeline
 name: pipeline1
 
 steps:
+- name: restore-cache
+  image: plugins/s3-cache
+  failure: ignore
```

Comment:
I've also marked all cache steps as optional. If there some credentials or S3 issues - it won't break all branches, instead dependencies will be downloaded as before